### PR TITLE
Adds exclude option to toBest 

### PR DESCRIPTION
**Pull request by NathanielScottStevens**
This is a clone of #32 (had some trouble with the old branch)

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -105,10 +105,14 @@ Converter.prototype.to = function (to) {
 /**
 * Converts the unit to the best available unit.
 */
-Converter.prototype.toBest = function() {
+Converter.prototype.toBest = function(options) {
   if(!this.origin)
     throw new Error('.toBest must be called after .from');
 
+  if (options === undefined) {
```
Comment (ben-ng) [inline lib/index.js]: I think that we can be more lenient and do `options == null` here.

File: lib/definitions/time.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
-var time
+var time;
+var daysInYear = 365.25;
```
Comment (ben-ng) [inline lib/definitions/time.js]: This looks like a breaking change that's unrelated to the rest of this pull request. Can you make the date changes a separate pull request so that we can release it as its own major version? Otherwise, people will have to upgrade to a new major version to get the exclude feature.

File: test/times.js [hunk 3]
```diff
@@ -27,10 +27,32 @@ tests['s to d'] = function () {
...
+    , actual = convert(52.17857).from('week').to('year');
+  assert.ok( percentError(expected, actual) < ACCURACY
+    , 'Expected: ' + expected +', Actual: ' + actual);
+};
 
-tests['w to year'] = function () {
-  assert.strictEqual( convert(52).from('week').to('year'), 1);
+tests['month to year'] = function () {
+  assert.strictEqual( convert(12).from('month').to('year'), 1);
 };
```
Comment (ben-ng) [inline test/times.js]: Thank you for adding new tests!

File: lib/definitions/time.js (hunk 2 above)
Comment (NathanielScottStevens) [inline lib/definitions/time.js]: These aren't my changes, they're from the merge with master. [8dd137b5a3fe68a1fb6db87fd74e4827ee516068](https://github.com/ben-ng/convert-units/commit/8dd137b5a3fe68a1fb6db87fd74e4827ee516068)
